packages/types/types/test.tsx

```diff
@@ -62,6 +68,53 @@ const myBlockLink = (
     dsRef={() => 'ima ref'}
   />
 )
+// $ExpectType Element
+const myButton = (
+  <Button
+    variation={'fill'}
+    size={'medium'}
+    color={'primary'}
+    disabled={false}
+    width={1}
+  >
+    BUTTON
+  </Button>
+)
+// $ExpectType Element
+const myCloseButton = (
+  <CloseButton
+    color={'background.darkest'}
+    onClick={() => 'ima function'}
+    title={'Title'}
+  />
+)
+// $ExpectType Element
+const myIcon = (
+  <Icon
+    name={'Email'}
+    size={48}
+    color={'primary'}
+    title={'Email'}
+    titleId={'Title ID'}
+    desc={'Description'}
+    descId={'Description ID'}
+  />
+)
+// $ExpectType Element
+const myIconButton = (
+  <IconButton
+    dsRef={() => 'ima ref'}
+    icon={<Icon name='Calendar' title='Choose date' />}
+    onClick={() => 'ima function'}
+  />
+)
+// $ExpectType Element
+const myIconField = (
+  <IconField>
```

Comment:
Also doesn't hurt to add some props to `IconField` for extra validation

Reply:
Added!